### PR TITLE
Add transaction log

### PR DESCRIPTION
**Pull request by jsmassa**

### CODE REVIEW

File: test/datahike/test/api.cljc [hunk 1]
```diff
@@ -672,3 +672,115 @@
                   hash-2 (hash @conn)]
               (is (not= hash-1 hash-2))
               (is (not= hash-0 hash-2)))))))))
+
+(defn test-write-schema [id cfg]
```
Comment (grischoun) [inline test/datahike/test/api.cljc:676]: These tests are not related to directly testing the introduction of `tx-log`, right? They are here to test that what existed before is not broken, right?

**Review by grischoun — APPROVED** [review]
Sounds great!
Good job!

**Comment by kordano** [comment]
will be implemented with #377
